charts/xrd-vrouter/values.yaml

```
@@ -256,6 +255,16 @@ cpu: {}
   #               * physical core 1 has logical cores 2 and 3
   #               * etc...
   #hyperThreadingMode: "off"
+  #
+  # Deprecated cpu settings:
+  #
+  # Override the cpuset to use for XRd. Must not be used in conjunction with
+  # controlPlaneCpuset and dataPlaneCpuset. (Deprecated.)
```

```suggestion
  # controlPlaneCpuset and dataPlaneCpuset.
  # Deprecated.  Use controlPlaneCpuset and dataPlaneCpuset instead.
```

It would be useful if the deprecation notice described what to use instead.